coordinator/core/src/main/java/io/stargate/core/metrics/impl/PrefixingMetricRegistry.java

```diff
@@ -108,7 +108,7 @@ public Timer timer(String name, MetricSupplier<Timer> supplier) {
 
   @Override
   @SuppressWarnings("rawtypes")
-  public Gauge gauge(String name, MetricSupplier<Gauge> supplier) {
+  public <T extends Gauge> T gauge(String name, MetricSupplier<T> supplier) {
```

Comment:
Signature change in "dropwizard-metrics" package: binary-compatible, source-incompatible